source/examples/xgboost-rf-gpu-cpu-benchmark/notebook.ipynb

```diff
@@ -0,0 +1,312 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "a51c95d1-b447-4f1b-9571-cf597ca93ef4",
+   "metadata": {},
+   "source": [
+    "<span style=\"display: block;  text-align: center; color:#8735fb; font-size:22pt\"> **HPO Benchmarking with RAPIDS and Dask** </span>\n",
+    "\n",
+    "Hyper-Parameter Optimization (HPO) helps to find the best version of a model by exploring the space of possible configurations. While generally desirable, this search is computationally expensive and time-consuming.\n",
+    "\n",
+    "In the notebook demo below, we compare benchmarking results to show how GPU can accelerate HPO tuning jobs relative to CPU.\n",
+    "\n",
+    "For instance, we find a x speedup in wall clock time and a x reduction in cost when comparing between GPU and CPU EC2 instances on 100 XGBoost HPO trials on 3 years of the Airline Dataset.\n"
```

Comment:
Can you put in the actual performance numbers for c5.24xlarge and p3.8xlarge ?

Reply:
I will update with the latest results once the cpu benchmarks are done. Merging for now.